tedana/tests/test_model_monoexponential.py

```diff
@@ -0,0 +1,43 @@
+"""
+Tests for tedana.model.monoexponential
+"""
+import os.path as op
+
+import numpy as np
+
+from tedana import utils
+from tedana.model import monoexponential as me
+from tedana.tests.utils import get_test_data_path
+
+
+def test_fit_decay():
+    tes = np.array([14.5, 38.5, 62.5])
+    in_files = [op.join(get_test_data_path(), 'echo{0}.nii.gz'.format(i+1))
+                for i in range(3)]
+    data, _ = utils.load_data(in_files, n_echos=len(tes))
+    mask, mask_sum = utils.make_adaptive_mask(data, minimum=False, getsum=True)
+    t2sv, s0v, t2ss, s0vs, t2svG, s0vG = me.fit_decay(data, tes, mask,
+                                                      mask_sum, start_echo=1)
+    assert len(t2sv.shape) == 1
+    assert len(s0v.shape) == 1
+    assert len(t2ss.shape) == 2
+    assert len(s0vs.shape) == 2
+    assert len(t2svG.shape) == 1
+    assert len(s0vG.shape) == 1
```

Comment:
This is partially preference, but I think `assert data.ndim == X` is a bit more parsimonious than `assert len(data.shape) == X`. I'm open to disagreement here, though! @emdupre do you have thoughts?

Reply:
I think you're probably right, @rmarkello, that `.ndim` is more elegant (and, importantly, I think equally easy to read !). Would you mind updating, @tsalo ?